lib/cs_guide/resources/venue.ex

```diff
@@ -12,8 +12,16 @@ defmodule CsGuide.Resources.Venue do
     field(:postcode, :string)
     field(:phone_number, :string)
     field(:cs_score, :float, default: 0.0)
+    field(:num_cocktails, :integer)
```

Comment:
`:num_cocktails` makes sense to be an :integer and let Ecto do the validation. 👍